src/components/post-block/style/lumapps/_index.scss

```diff
@@ -54,13 +54,28 @@
 
         display: block;
         margin-bottom: $lumx-spacing-unit;
-        color: lumx-theme-color-variant('dark', 'L2');
+
+        #{$self}--theme-light & {
+            color: lumx-link('dark', 'L2');
+        }
+
+        #{$self}--theme-dark & {
+            @include lumx-link('light', 'L2');
+        }
```

Comment:
Sorry I missed in the first review, but please use the `lumx-theme-color-variant` instead
```scss
        #{$self}--theme-light & {
            color: lumx-theme-color-variant('dark', 'L2');
        }

        #{$self}--theme-dark & {
            color: lumx-theme-color-variant('light', 'L2');
        }
```
The `lumx-link` mixin is really for <a> links